Uplift/Views/Cells/FacilitiesDropdownCollectionViewCell.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  FacilitiesDropdownCollectionViewCell.swift
+//  
+//
+//  Created by Cameron Hamidi on 11/6/19.
+//
+
+import UIKit
+
+class FacilitiesDropdownCollectionViewCell: UICollectionViewCell {
+    var collectionView: UICollectionView!
+    var collectionViewHeight: CGFloat!
+    var dropdownView: DropdownView!
+    var facility: Facility!
+    var headerImageView: UIImageView!
+    var headerNameLabel: UILabel!
+    var headerOpenLabel: UILabel!
+    var headerView: UIView!
+    var headerViewHeight: CGFloat = 52
+    var height: CGFloat!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupDropdownView()
+        setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func configure(for facility: Facility) {
+        self.facility = facility
+        setupDropdownView()
+        setupConstraints()
+    }
+    
+    func setupDropdownView() {
+        headerView = setupHeaderView(headerString: facility.name.uppercased())
+        
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: StretchyHeaderLayout())
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        //TODO: register facility cell classes once they're finished
+        
+        collectionViewHeight = 0 //TODO: once cell classes are finished, sum up their heights to get the total collection view height
+        
+        dropdownView = DropdownView(delegate: self,
+                                    headerView: headerView,
+                                    headerViewHeight: headerViewHeight,
+                                    contentView: collectionView,
+                                    contentViewHeight: collectionViewHeight)
+    }
+    
+    func setupHeaderView(headerString: String) -> UIView {
+        var headerImage: UIImage
+        switch headerString {
+        case "EQUIPMENT", "FITNESS CENTER":
+            headerImage = UIImage(named: ImageNames.equipment)!
+        case "GYMNASIUM":
+            headerImage = UIImage(named: ImageNames.basketball)!
+        case "SWIMMING POOL":
+            headerImage = UIImage(named: ImageNames.pool)!
+        case "BOWLING LANES":
+            headerImage = UIImage(named: ImageNames.bowling)!
+        default:
+            headerImage = UIImage(named: ImageNames.misc)!
+        }
```

Comment:
can we use an enum for this instead of just comparing string values? you could do something like this:
```
private enum FacilityName: String {
  case equipment = "EQUIPMENT"
  case gymnasium = "GYNMASIUM"
  ...
}
```
and then create an enum w/ the facility name by doing something like: `FacilityName(rawValue: facility.name.uppercased())`.
another question I have though is why we are uppercase'ing them?

Reply:
also I believe we'll never need a header with "FITNESS CENTER"?

Reply:
@kevinchan159 we're uppercasing them because the designs specify that these labels are all uppercased.

@ywsang currently backend uses the term "Fitness Center" instead of "Equipment".